### ISSUE TITLE
[Feature request] Download and Install packages in parallel

### ISSUE THREAD

**Issue by marceloatie** [openSUSE/zypper#160]
A new option like 'zypper up --download-in-parallel' would be great.

It will download all packages and install then in parallel

'zypper up --download-as-needed' works very close to it, but it's not in parallel.

**Comment by Tellack** [comment]
Yes, I totally agree with this advice. Many packages are so small that we lose even much more time on establishing the Internet connections rather than on downloading the metadata. 

So zypper should make an option of downloading packages by multiple threads, download more than one package at the same time, like dnf does. 

This can really save users' time.

**Comment by marceloatie** [comment]
i mean one thread for downloading and one thread for installing, but after that i think will be easy to implement multiple thread for downloading (in further).

**Comment by shasheene** [comment]
Note: this issue is a duplicate of the older issue #104 (parallel download), and the closed issue #154 (parallel install).

**Comment by shasheene** [comment]
For thoroughness, linking to a newer (though closed) issue #404, requesting a parallel install feature.

**Comment by romulasry** [comment]
Update?

**Comment by bzeller** [comment]
For the download part this is already tracked in #104 we are working on that, as for the install in parallel part this depends completely on support from rpm and is out of our hands. Closing this issue, please follow #104 for updates.
